sport/app/services/dotcomrendering/FootballPagePicker.scala

```diff
@@ -9,7 +9,9 @@ import utils.DotcomponentsLogger
 object FootballPagePicker {
 
   def isSupportedInDcr(page: FootballPage): Boolean = {
-    page.metadata.id == "football/live"
+    val footballMatchesPattern =
+      """^football(?:/[^/]+)?/(live|fixtures|results)(?:/more)?(?:/\d{4}/[A-Za-z]{3}/\d{1,2})?$""".r
```

Comment:
How does this bit work? `(?:/[^/]+)?`

Reply:
When we use `()` it create a capturing group, meaning whatever matches inside will be stored and can be accessed later. But `(?: ... )` is a non-capturing group, it groups the pattern for logic but doesn't store the match, making the regex more efficient. 

There's some explanation about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Non-capturing_group